File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/non-plagiarized/04/T02.kt
package IR

import java.text.DecimalFormat
import java.util.Scanner

fun main() {
    val s = Scanner(System.`in`)
    println("Enter the radius and length of a cylinder :")
    val radius = s.nextDouble()
    val length = s.nextDouble()
    val area = radius * radius * 3.14159

    val df = DecimalFormat(".####")
    val df2 = DecimalFormat(".#")
    val volume = area * length
    println("The area is ${df.format(area)}")
    println("The volume is ${df2.format(volume)}")
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/non-plagiarized/04/T02.ts
import { DecimalFormat } from 'java.text';
import { Scanner } from 'java.util';

function main(args: string[]): void {
    const s: Scanner = new Scanner(System.in);
    console.log("Enter the radius and length of a cylinder :");
    const radius: number = s.nextDouble();
    const length: number = s.nextDouble();
    const area: number = radius * radius * 3.14159;

    const df: DecimalFormat = new DecimalFormat(".####");
    const df2: DecimalFormat = new DecimalFormat(".#");
    const volume: number = area * length;
    console.log("The area is " + df.format(area));
    console.log("The volume is " + df2.format(volume));
}

